main.go

```diff
@@ -229,6 +229,9 @@ func init() {
 }
 
 func dumpFSM(mapper *mapper.MetricMapper, dumpFilename string, logger log.Logger) error {
+	if mapper.FSM == nil {
+		return fmt.Errorf("FSM is not available. Posible due to the presence of regex patterns")
```

Comment:
I'm not a fan of "Possibly X" errors, I think we can do better in this case. Rather than saying "maybe" here I would prefer to detect this combination of flags in main(), emit a specific warning, and not call `dumpFSM` at all.

Reply:
Sounds fine. I'll look into it. 
Right now there's no way of knowing if the FSM availability is because of a regex as the mapping library does not make it possible to know with any exported method. Do you think it makes sense to add that method there and then use it in this project for a check?

Reply:
ah you're right, I misunderstood. I thought regexes would somehow be in flags but of course they're only in the mapping file itself … in that case let's leave it as it is here. I would like to change the wording a little bit:

```suggestion
		return fmt.Errorf("no FSM available for to be dumped, possibly because the mapping contains regex patterns")
```

Reply:
Sorry, I had to add a signoff and had to force push.